src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Teams/TeamNameViewModel.cs

```diff
@@ -58,6 +66,19 @@ public static TeamNameViewModel Map(Team team)
             };
         }
 
+        /// <summary>
+        /// Returns team's group name
+        /// </summary>
+        /// <param name="team">Mapped team</param>
+        /// <param name="divisions">Divisions list in the current tournament</param>
+        /// <returns>String which contains group name</returns>
+        public static string GetGroupName(TeamNameViewModel team, List<Division> divisions)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to use a generic collection designed for inheritance. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3956)